components/style/stylist.rs

```diff
@@ -458,6 +471,38 @@ impl Stylist {
             .map(|(d, _)| d.invalidation_map.len()).sum()
     }
 
+    /// Set the registered property set associated with the document this
+    /// Stylist is styling.
+    pub fn set_registered_property_set(
+        &mut self,
+        registered_property_set: Arc<Locked<RegisteredPropertySet>>
+    ) {
+        debug_assert!(if let Some(ref old) = self.registered_property_set {
```

Comment:
nit: Again, `as_ref().map_or(true, ...)` instead of `if let (..) { } else { .. }`.

Reply:
Nice catch!